packages/notifi-dapp-example/src/components/DashboardSideBar.tsx

```diff
@@ -83,7 +83,7 @@ export const DashboardSideBar: FC<DashboardSideBarProps> = ({
                 cardView === 'alertSubscription'
                   ? 'text-notifi-icon-selected'
                   : 'text-notifi-icon-unselected'
-              }`}
+              } mt-[2px]`}
```

Comment:
We better use "rem" instead of 'px' so that it can auto adjust when users zoom in/out the browser.